app/server/src/routes/userProfile/userProfile.controller.js

```diff
@@ -0,0 +1,59 @@
+const userProfileModel = require("../../models/userProfile/userProfile.model");
+const personalInfoModel = require("../../models/personalInfo/personalInfo.model");
+const badgeModel = require("../../models/badges/badges.model");
+const mongoose = require("mongoose");
+
+const UserProfileController = {
+    
+    postCreate: async function (req,res) {
```

Comment:
I think since we integrated userprofile data to user model, we can convert this function to updatePersonalInfo.
It could basically get bio string and update the personalInfo.bio for now.

```jsx
 
// POST
updatePersonalInfo({
  'user_id': '634cf129a727c14722a631fa',
  'bio': 'Hi! I am Muhammet. I am a junior computer engineering student. I have several projects and jobs going on. I am interested in backend development and DevOps',
 
}) -> 
{
  'message': 'Profile is updated.' 
}
```

Reply:
Changed the endpoint, it now updates the bio of the user